### ISSUE TITLE
List on Header missing unique keys

### ISSUE THREAD

**Issue by MikelTafalla** [bcantello/peoples_voice_project#57]
I believe it is the <a> tag list created at header.js when we push the tag into registerToVote. The tag needs to have the 'key' attribute and it needs to be unique
![list_unique_keys](https://user-images.githubusercontent.com/61996656/225483081-48bc40d6-4492-4b52-9ee1-ddeb509e550a.png)

**Comment by bcantello** [comment]
Yeah. We could probably just add some logic to add a number as the key. Just count up from 0 or 1.
